pagopa/pn-frontend
# PN-13439 - Show informative dialog on courtesy contact activation

## Short description
Add the `InformativeDialog` component, which is displayed when attempting to add a courtesy contact (email, SMS, or AppIO) while the SERCQ SEND feature is not enabled. For the AppIO case, the dialog is always shown (also with the SERCQ SEND enabled), but the content of the text changes slightly.

## List of changes proposed in this pull request

- Implement the `InformativeDialog` component
- Display the `InformativeDialog` during the activation of courtesy contacts

## How to test
Start PF/PG and attempt to add an email or SMS contact or to activate the AppIO service (only for PF). There are two possible scenarios:

1. SERCQ SEND Enabled
- For EMAIL or SMS: you should not see any modal when trying to add these contacts.
- For AppIO: you should see the dialog, but without any content text. [Figma](https://www.figma.com/design/DlzvNQzkecy7L2ym40OTge/Domicilio-digitale---08%2F11?node-id=2639-81122&t=c5n6rXZKQiGHwelH-4)

2. SERCQ SEND Not Enabled
- For EMAIL or SMS: you should see the `InformativeDialog`
- For AppIO: you should see the dialog with the content text (Se non hai il domicilio digitale...). [Figma](https://www.figma.com/design/DlzvNQzkecy7L2ym40OTge/Domicilio-digitale---08%2F11?node-id=2698-19057&t=c5n6rXZKQiGHwelH-4)
